src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -68,6 +71,9 @@ public enum ErrorCode {
 
   UNSUPPORTED_UPDATE_OPERATION("Unsupported update operation"),
 
+  UNSUPPORTED_COMMAND_EMBEDDING_SERVICE(
+      "Unsupported command createEmbeddingService since application is configured for property based embedding"),
```

Comment:
Include (single) quotes around createEmbeddingService -> 'createEmbeddingService' to be consistent with other messages.